app/containers/MessageBox/Recording.js

```diff
@@ -102,7 +105,8 @@ export default class extends React.PureComponent {
 			this.recording = false;
 			const filePath = await AudioRecorder.stopRecording();
 			if (isAndroid) {
-				this.finishRecording(true, filePath);
+				const data = await RNFetchBlob.fs.stat(decodeURIComponent(filePath));
```

Comment:
I think here is better to use `FileSystem.getInfoAsync(fileUri, options)`, I can't use it on Share Extension yet that's why I use `RNFetchBlob` but this code not is on Share Extension then you can use `Expo-File-System`.